src/views/Home/Filters/index.tsx

```diff
@@ -89,7 +89,7 @@ function Filters(props: Props) {
             <MultiSelectInput
                 label={strings.alertUrgency}
                 placeholder={strings.alertUrgency}
-                name="urgencyList"
+                name={strings.alertUrgencyListName}
```

Comment:
We shouldn't use translated strings for the variable here. This `name` variable is used internally by the component.